apps/ledger-live-desktop/src/renderer/families/evm/AccountHeaderManageActions.ts

```diff
@@ -32,7 +32,12 @@ const AccountHeaderActions = ({ account, parentAccount }: Props) => {
     }
   }, [account, dispatch, parentAccount]);
 
-  if (account.type === "Account" && account.currency.id.includes("ethereum")) {
+  if (
+    account.type === "Account" &&
+    // Ethereum Classic is still on proof of work
+    account.currency.id !== "ethereum_classic" &&
+    account.currency.id.includes("ethereum")
```

Comment:
```suggestion
    account.currency.id === "ethereum"
```
This is the alternative, if we do _not_ want to include  any of:
 "ethereum_ropsten",
  "ethereum_goerli",
  "ethereum_sepolia",
  "ethereum_holesky",

Reply:
If we support staking on ethereum only then it should be a strict "===" on ethereum imo